### DISCUSSION TITLE
how to use .redact to redact only certain TYPES like person or other entiity

### DISCUSSION THREAD

**Discussion by vvonchain** [microsoft/presidio#1324] [General]
title

**Comment by omri374** [comment]
Could you please clarify? Is this for text out images?

**Reply by vvonchain** [reply]
images
`redact(image, fill='contrast', padding_width=25, crop_ratio=0.75, type='PERSON')`
it  should redact only person names

**Reply by omri374** [reply]
You can pass an `entities` field:
`redact(image, fill='contrast', padding_width=25, crop_ratio=0.75, entities=["PERSON", "URL"])`
